CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Preliminary support for BINARY and VARBINARY SQL
+  data types.
```

Comment:
Link the PR

Reply:
In general you cannot write a link to a PR in a CHANGELOG before you have created the PR